kubernetes/config-maps/nginx-proxy/default.conf.j2

```diff
@@ -25,6 +25,12 @@ server {
     listen 443      ssl http2 default_server;
     listen [::]:443 ssl http2 default_server;
 
+    ssl_certificate     /etc/ssl/secrets/nginx-proxy-ssl.crt;
+    ssl_certificate_key /etc/ssl/secrets/nginx-proxy-ssl.key;
+
+    # Add Diffie-Helman group
+    ssl_dhparam /etc/ssl/secrets/nginx-proxy-dhparam.pem;
```

Comment:
How big is this file and how long does it take to generate it?

It would be best to generate it during container build.

Reply:
It's a small file which is generate fast. This solution requires additional script and causes generate new `.pem` file for new deployment.

Reply:
Are you sure we want to use it ?

Reply:
Why not?

Reply:
Why want to it? Ok, so I will do it.

Reply:
I checked how much time need to generate this file and:
- `~10s for 2048bit`
- `~30s for 4096bit`

Reply:
We still want this solution ?

Reply:
I checked, how much time need to generate this file and it's not deterministic and random on each execution. For generate 2048bit key, it's need sometimes 10s and sometimes 40s. The openssl inform that `Generating DH parameters, 2048 bit long safe prime, generator 2
This is going to take a long time`. We still want this solution ?